BioForm/profiles/static/scripts/register.js

```diff
@@ -0,0 +1,158 @@
+const form = document.getElementById("form");
+const username = document.getElementById("username");
+const first_name = document.getElementById("first-name");
```

Comment:
Unlike Python, JavaScript uses camelCase naming convention for variables and functions and classes are PascalCase. Make this change in all places.
```suggestion
const firstName = document.getElementById("first-name");
```